stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -239,6 +247,462 @@ lazy_static! {
     ADD COLUMN burn_view TEXT;
     "#.into(),
     ];
+
+    pub static ref NAKAMOTO_CHAINSTATE_SCHEMA_3: Vec<String> = vec![
+    NAKAMOTO_TENURES_SCHEMA_3.into(),
+    r#"
+    UPDATE db_config SET version = "6";
+    "#.into(),
+        // Add a `height_in_tenure` field to the block header row, so we know how high this block is
+        // within its tenure.  This is needed to process malleablized Nakamoto blocks with the same
+        // height, as well as accidental forks that can arise from slow miners.
+        //
+        //
+        //
+        // No default value is needed because at the time of this writing, this table is actually empty.
+    r#"
+    ALTER TABLE nakamoto_block_headers
+    ADD COLUMN height_in_tenure;
+    "#.into(),
+    ];
+}
+
+/// MARF key/value pairs
+pub mod nakamoto_keys {
+    use super::{hex_bytes, to_hex, ConsensusHash, NakamotoTenureId, StacksBlockId};
+
+    pub fn ongoing_tenure_id() -> &'static str {
```

Comment:
These methods all need rustdocs explaining what the key is used to store.

Reply:
Sure